setup.py

```diff
@@ -51,7 +51,12 @@ def find_version(*file_paths):
     package_data={"PyQt5-stubs": ['*.pyi']},
     packages=["PyQt5-stubs"],
     extras_require={
-        "dev": ["mypy==0.991", "pytest", "pytest-xvfb"],
+        "dev": [
+            "mypy==0.991; python_version >= '3.7'",
+            "mypy==0.930; python_version < '3.7'",
+            "pytest",
+            "pytest-xvfb",
+        ],
     },
```

Comment:
I did not know you can do things like this. Nice!

Reply:
See the `env_var` list at https://peps.python.org/pep-0508/.

```
env_var       = ('python_version' | 'python_full_version' |
                 'os_name' | 'sys_platform' | 'platform_release' |
                 'platform_system' | 'platform_version' |
                 'platform_machine' | 'platform_python_implementation' |
                 'implementation_name' | 'implementation_version' |
                 'extra' # ONLY when defined by a containing layer
                 )
```

Though it appears this boundary may belong closer to 0.981 https://github.com/python/mypy/commit/dc118e293203863ab1007699b2cecf0f26ddfa22.